Readme.md

```diff
@@ -164,14 +164,15 @@ $ = cheerio.load('<ul id="fruits">...</ul>', {
 });
 ```
 
-These parsing options are taken directly from [htmlparser2](https://github.com/fb55/htmlparser2/wiki/Parser-options), therefore any options that can be used in `htmlparser2` are valid in cheerio as well. The default options are:
+These parsing options are taken directly from [htmlparser2](https://github.com/fb55/htmlparser2/wiki/Parser-options), therefore any options that can be used in `htmlparser2` are valid in cheerio as well. If any of these options is set to non-default value cheerio will implicitly use `htmlparser2` as an underlying parser. In addition, you can use `useHtmlParser2` option to force cheerio use `htmlparser2` instead of `parse5`. The default options are:
 
 ```js
 {
     withDomLvl1: true,
```

Comment:
I had the same question, so [I experimented a bit](https://gist.github.com/jugglinmike/3661b713ffb0393c5dba16375255389d) and found that Dom Level 1 support is analogous. [Here's the relevant source code](https://github.com/inikulin/parse5/blob/969ed22f53db38c101449e05a1b804ea456d4e04/lib/tree_adapters/htmlparser2.js#L7-L62). Great!